CMakeCache.txt

```diff
@@ -0,0 +1,38 @@
+# This is the CMakeCache file.
```

Comment:
I am sure you do not want to push this to master...

Reply:
I mean the whole file..

Reply:
Good catch!